installation/automated.md

```diff
@@ -29,9 +27,8 @@ The automated installer provides the following:
 * An offline version of our [demos](https://demos.telerik.com/blazor-ui/) that you can run and inspect in your IDE. You can find them in the `demos` folder in the installation. @[template](/_contentTemplates/common/get-started.md#demos-project-net-version)
 * The [document processing]({%slug common-features-dpl%}) `.nupkg` files (in the `dpl` folder).
 * A shortcut to our [online demos](https://demos.telerik.com/blazor-ui/).
-* Our [Visual Studio Extensions]({%slug getting-started-vs-integration-overview%}).
-    * We also have [extensions for VS Code]({%slug getting-started-vs-code-integration-overview%}) and you can install them from its own marketplace (<a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard" target="_blank">direct link</a>).
-
+* Our [Visual Studio extension]({%slug getting-started-vs-integration-overview%}).
+    * We also have an [extension for VS Code]({%slug getting-started-vs-code-integration-overview%}) and you can <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard" target="_blank">install it from the VS Code marketplace</a>.
```

Comment:
A single-item nested list looks weird and does not really serve a purpose. This might work better as a nested tip, or a new line within the bullet. You can also incorporate it in the upper-level bullet or move it as its own bullet on the same level as the VS one.

Reply:
Cannot be a separate item in the list, because it's not part of the installer. I merged it with its parent item.